Sources/Stencil/Variable.swift

```diff
@@ -111,12 +111,12 @@ public struct Variable: Equatable, Resolvable {
     } else if let string = context as? String {
       return resolve(bit: bit, collection: string)
     } else if let object = context as? NSObject {  // NSKeyValueCoding
-      #if os(Linux)
-        return nil
-      #else
+      #if _runtime(_ObjC)
```

Comment:
As discussed in #341, I don't there's need for this, and we can just use `canImport` for now.

Reply:
Yeah, that commit needs to be dropped, I couldn't easily do so though.